packages/core/core/src/Environment.js

```diff
@@ -88,6 +89,9 @@ export function createEnvironment({
       case 'electron-renderer':
         outputFormat = 'commonjs';
         break;
+      case 'edge':
+        outputFormat = 'esmodule';
+        break;
```

Comment:
At least Cloudflare Workers and Vercel Edge Runtime only support ESM

Reply:
Hey, is that possible to have an option to generate a single commonjs bundle as well?

Reply:
This is just the default, you can always do
```
{
  "targets": {
    "default": {
      "context": "edge",
      "outputFormat": "commonjs"
    }
  }
}
```
(and as I've said, as far as I can tell most of these runtimes only support ESM, which is why I made that the default.).

By "single" you mean making dynamic imports behave like a synchronous require instead? I think that's out of scope for this PR (and also there have been people in the past who wanted this feature for Node as well)